static/docs/user-guide/update-DVC-pipelines.md

```diff
@@ -1,34 +1,50 @@
 # Updating pipelines
 
-If you need to rename or relocate the file paths and/or dependencies of a
-specific DVC-file, then you can 
+If you need to rename or relocate the file paths and/or dependencies and
```

Comment:
`relocate the file paths` looks weird. It should be probably just `relocate a DVC-file and/or ... of the DVC-file`